Kitodo/src/main/java/de/sub/goobi/forms/HelperForm.java

```diff
@@ -304,4 +305,13 @@ public String getUserAgent() {
     public boolean getUserStepDoneSearchEnabled() {
         return ConfigCore.getBooleanParameter("withUserStepDoneSearch");
     }
+    public static void saveMessage() {
+
+
+        FacesContext context = FacesContext.getCurrentInstance();
+
+        context.addMessage(null, new FacesMessage(FacesMessage.SEVERITY_ERROR,"Failed",  "The elastic search server is not running!") );
```

Comment:
1) Please fix formatting of method parameters.
2) This kind of message is not translatable - is this intended?